### PR TITLE
feat: add firmware update for Inovelli LZW31-SN

### PR DESCRIPTION
**Pull request by InovelliUSA**
This is an initial device commit to make sure the formatting is correct and that everything is in order.

### CODE REVIEW

File: firmwares/inovelli/LZW31-SN.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+	"devices": [
+		{
+			"brand": "Inovelli",
+			"model": "LZW31-SN",
+			"manufacturerId": "0x031E",
```
[the remark this answers is not in the document]
Comment (AlCalzone) [reply firmwares/inovelli/LZW31-SN.json:6]: Other than that seems to be okay. I'll rebase your PR so the downloads and integrity hashes can automatically be checked.

**Comment by AlCalzone** [comment]
@InovelliUSA let me know if you want to add other files to this PR or if I should merge this

**Comment by InovelliUSA** [comment]
Can we merge this product by itself? I'd like to check the process and make sure everything is in good order and then I will submit the rest of the products in a different PR.

**Comment by AlCalzone** [comment]
Sure thing!

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
This should be live in a few minutes up to an hour.

**Pull request merged**